l10n_it_ricevute_bancarie/migrations/12.0.1.6.1/post-migrate.py

```diff
@@ -0,0 +1,13 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        invoice_ids = env['account.invoice'].search([
```

Comment:
trattandosi di un recordset puoi rinominarlo in `invoices`?

Reply:
fatto